packages/cli-repl/src/cli-repl.ts

```diff
@@ -298,11 +298,12 @@ class CliRepl {
     // This checks for error instances.
     // The writer gets called immediately by the internal `this.repl.eval`
     // in case of errors.
-    if (result && result.message && typeof result.stack === 'string') {
-      this.bus.emit('mongosh:error', result);
+    if (result && (result.message || result.errmsg)) {
```

Comment:
if that's the case probably would have been great to have this in a test

Reply:
Yeah, that would pass. I was trying to figure out how to actually determine if something is an error, but the overlap between driver errors (specifically the bulk one) and regular errors is almost nothing. The driver error looks like:
```
{
  index: 0,
  code: 9,
  errmsg: 'multi update is not supported for replacement-style update',
  op: { q: {}, u: { x: 2 }, multi: true, upsert: false }
}
```
While the JS error has `name`, `stack`, `message`, and `code`. I suppose I could be more exclusive and check for exactly the driver error format or exactly the js error? It seems odd to me that we'd ever be in a situation where we have to determine if something is an error via its attributes, it seems like it should be a different code path. Maybe I should see if I can refactor a little to avoid the problem entirely?

Reply:
Ah, it's a node repl thing. Why anyone thought that was a good idea is a little mystifying, oh well. I'll check for either/or instead of something that works for both.